File: pingyougou-user-web/src/main/java/com/pinyougou/user/controller/LoginController.java
package com.pinyougou.user.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/login")
public class LoginController {

    /**
     * 返回登录名
     * @return
     */
    @RequestMapping("/getLoginName")
    public Map getLoginName(){
        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        Map map=new HashMap();
        map.put("loginName",name);
        return map;
    }
}

File: pingyougou-user-web/src/main/webapp/js/controller/loginController.js
//控制层
app.controller('loginController' ,function($scope,$controller,loginService) {
    //获取登录名
    $scope.getLoginName = function () {
        loginService.getLoginName().success(
            function (response) {
                $scope.loginName = response.loginName;
            }
        );
    }
});